packages/js/src/Modules/Verto/webrtc/BaseCall.ts

```diff
@@ -414,6 +414,28 @@ export default abstract class BaseCall implements IWebRTCCall {
     toggleAudioTracks(this.options.localStream);
   }
 
+  /**
+   * Changes the audio input device (i.e. microphone) used for the call.
+   *
+   * @examples
+   *
+   * Using async/await:
+   *
+   * ```js
+   * await call.setAudioInDevice('abc123')
+   * ```
```

Comment:
I think this could be a better usage example if we showed how you get the device ID from `getAudioInDevices` and use it with `setAudioInDevice`.